boards/arm/stm32f411e_disco/stm32f411e_disco.dts

```diff
@@ -79,3 +79,7 @@
 &rtc {
 	status = "okay";
 };
+
+usbd: &usbotg_fs {
```

Comment:
Sory for the back and force comment on gthis, but please remove alternate label `usbd: ` for now

Reply:
No problem at all. Thanks for the guidance. I have reverted it to the original implementation.